### ISSUE TITLE
SPRING Data Transfer Request for Inactive Volunteers

### ISSUE THREAD

**Issue by iverkaik** [BiologicalRecordsCentre/SPRING#123]
Hi,

I have two volunteers (associated to two sites) who are no longer associated with the institution where they entered data. Is it possible to transfer the data they entered to another user? This is to ensure continued access to the species list.

Thank you,
Iraima

**Comment by DavidRoy** [comment]
@iverkaik yes, please provide details of the change needed. By email if the information is sensitive (i.e. don't inlcude email addresses in a github thread)

**Comment by iverkaik** [comment]
excellent, so I will do!

thank you @DavidRoy
